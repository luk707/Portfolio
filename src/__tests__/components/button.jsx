// @flow

import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider } from 'emotion-theming'
import Button from '../../components/button'
import defaultTheme from '../../util/theme'

describe('card', () => {
  it('renders as expected', () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={defaultTheme}>
          <Button>Hello world</Button>
        </ThemeProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
