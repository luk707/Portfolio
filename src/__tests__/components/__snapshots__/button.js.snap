// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card renders as expected 1`] = `
<button
  className="css-by0ps9"
>
  Hello world
</button>
`;
